python/clean_export_aggregate.py

```diff
@@ -0,0 +1,53 @@
+import pandas as pd
+import sys
+from sqlalchemy import create_engine
+
+BUILD_ENGINE = sys.argv[1]
```

Comment:
Would it cleaner to just read this from the local `.env` instead of passing it as a parameter?

Reply:
good point. I was thinking about how it is used on github actions. Open to suggestions on make it cleaner

Reply:
It will still have access to .env on github actions, that's what [these lines](https://github.com/NYCPlanning/db-developments/blob/8e08818698aeff004a9dc29f6e54be11186a16f5/.github/workflows/publish.yml#L13) are for

Reply:
Oh sorry I forgot about this comment, can we get the build engine with `os.environ`?

Reply:
Hey sorry @td928 once we resolve this we can merge. I had to comment this line out when running the code in the ipython shell, which I why I thought we should get it straight from .env. There's no time we would pass another build engine, right?